CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+- SAMPLE and SAMPLE_GROUP work consistently with seed in RSpec. ([@stefkin][])
```

Comment:
Please, add a shortcut with the URL to the bottom of the file. Currently, it's not clickable: https://github.com/palkan/test-prof/blob/903649336e49ec042aa85d6a8530499396ae14a0/CHANGELOG.md

Reply:
done, thanks